datasets/esa-worldcover.yaml

```diff
@@ -29,25 +30,48 @@ Resources:
     Region: eu-central-1
     Type: S3 Bucket
     RequesterPays: False
+    Explore:
+    - '[STAC endpoint](https://services.terrascope.be/stac/)'
 DataAtWork:
   Tutorials:
+  - Title: Exploring the dataset (STAC API examples)
+    URL: https://github.com/ESA-WorldCover/esa-worldcover-datasets
```

Comment:
@dzanaga can you make the github repo public if you want to use it for a documentation link?

Reply:
Done